master/reference/public-cloud/aws.md

```diff
@@ -38,85 +20,62 @@ on your deployment.
 Calico requires the following security group exceptions to function properly
 in AWS.
 
-| Description    | Type            | Protocol | Port Range |
-|:---------------|:----------------|:---------|:-----------|
-| BGP            | Custom TCP Rule | TCP      | 179        |
+| Description      | Type            | Protocol | Port Range |
+|:-----------------|:----------------|:---------|:-----------|
+| BGP              | Custom TCP Rule | TCP      | 179        |
 | \*IPIP           | Custom Protocol | IPIP     | all        |
 
->\*IPIP: Only required if using Calico with IPIP encapsulation.
+>\*IPIP: This rule is required only when using Calico with IPIP encapsulation.
 Keep reading for information on when IPIP is required in AWS.
 
-You can check if your hosts have successfully established BGP sessions with
-one another using `calicoctl node status`.
-
-#### Allow Traffic Within a Single VPC Subnet
-
-By default, each EC2 instance performs source/destination checks on inbound
-and outbound traffic. This ensures that the instance is the source or
-destination of any traffic it sends or receives.
+#### Allowing Traffic Within a Single VPC Subnet
 
-Since Calico allocates a unique IP address to each container that isn't the IP
-of its host, and since Calico doesn't encapsulate container traffic by default,
-traffic to/from containers will be dropped by the AWS src/dst checks
+To use Calico within a single VPC subnet you will need to _disable_ AWS src/dst
+checks on each EC2 instance in your cluster
+[as described in the AWS documentation](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck).
 
-This reverse-path filtering is enabled by default, but can be disabled on each
-host, allowing Calico traffic to pass through unaffected **between hosts within
-the same VPC subnet.**
-Ensure you follow
-[AWS documentation on disabling source/destination checks](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck).
+When src/dst checking is enabled, the AWS fabric enforces reverse path filtering which
+drops traffic between containers in the cluster after it leaves the host EC2 instance.
 
-#### Allow Traffic Across Different VPC Subnets
+#### Allowing Traffic Across Different VPC Subnets / VPCs
 
-Container traffic that traverses different VPC subnets will pass through
-a VPC routing table. If this VPC routing table does not know how to route
-container/pod IPs, it will drop the traffic. There are two ways two prevent this from
-happening:
+To use Calico across multiple different VPC subnets or [peered VPCs](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-peering.html),
+in addition to disabling src/dst checks as described above you must also enable ipip encapsulation and outgoing NAT
```

Comment:
OK, makes sense.  Maybe split it into a second sentence then since it's pretty non-obvious what function this gains you.  But then you could argue the first half of the sentence should have a why.  And then maybe the whole thing gets too long.